AWS/legos/aws_get_secrets_manager_secretARN/aws_get_secrets_manager_secretARN.py

```diff
@@ -1,44 +1,25 @@
-
 ##
 ##  Copyright (c) 2023 unSkript, Inc
 ##  All rights reserved.
 ##
-
 from __future__ import annotations
-
 from typing import Optional
-
-from pydantic import BaseModel, Field
-
-
-class InputSchema(BaseModel):
```

Comment:
why is the input schema being deleted?

Reply:
That was my mistake I think I forgot to add. Now it's updated.